### DISCUSSION TITLE
Errors on unused livewire components

### DISCUSSION THREAD

**Discussion by jcc5018** [livewire/livewire#4950] [1. Help]
I had a few livewire components created from packages that I no longer have. When loading a page, I was getting errors about classes not being found for these unused components.  

Why are errors appearing from components that aren't being used or called?  If I am experimenting with a component or still in the process of creating something, I don't need that preventing me from loading completely unrelated pages.

**Comment by joshhanley** [comment]
@jcc5018 can you provide a screenshot of the errors so we can investigate?

Thanks!

**Comment by jcc5018** [comment]
So I had fixed the issue by removing the unused livewire elements that were throwing errors, but after restoring them, i am not currently seeing the errors to screen shot. But the gist of it was it was not finding the reference to  PowerGridComponent or the FormComponent, both of which were packages I installed at one time. 

Bastinald\LaravelLivewireForms and PowerComponents\LivewirePowerGrid

But neither of those packages or components were being called from the page I was trying to access as far as I could tell, yet they were preventing me from loading the page.  

If it happens again, i will try to provide a screen shot.
